README.md

```diff
@@ -1,10 +1,10 @@
 # identity-api-server
 
-|  Branch | Build Status | Travis CI Status |
```

Comment:
I am not sure what this removed status is or if we are currently using it. It's better to validate this with someone who has knowledge about it since the framework repository also has this status.

Reply:
We don't use travis CI anymore afaik. I think we need to remove it from the framework repo as well. WDYT?

cc: @AnuradhaSK